intranet/apps/preferences/fields.py

```diff
@@ -25,7 +22,5 @@ def to_python(self, value):
         return value
 
     def widget_attrs(self, widget):
-        attrs = {}
-        if self.max_length is not None:
-            attrs.update({'maxlength': str(self.max_length)})
+        attrs = {"maxlength": "17"}
```

Comment:
@ezwang Yes, that's correct, according to the code linked above, it should be set to None.